### ISSUE TITLE
--yarn does not create start script

### ISSUE THREAD

**Issue by sumanthratna** [preactjs/preact-cli#1179]
# Bug

## Current behaviour
Running `yarn start` doesn't work:
```
yarn run v1.22.4
error Command "start" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

## Steps to reproduce
```zsh
preact create typescript myproj --yarn
cd myproj
yarn start
```

## Expected behaviour
`yarn start` should run the development server.

## Other relevant information
- macOS 10.15.5 Beta 4
- npm v6.14.4
- preact-cli 2.2.1
- yarn run v1.22.4

## `preact info`:
```
         ▄▄
     ▄▄▓▓▓▓▓▓▄▄
  ▄█▀▀█▓▓▓▓▓▓▓▀▀█▄▄
▐▓▌▐▓▓▓▒▄ ▀▄▄▓▓▓▌▐▓▌ 
▐▓▓▄▀▓▀ ▄▓▓▄▄▀▓▓ ▓▓▌ 
▐▓▓▓▌ ▒▓▌  ▐▓▓  ▓▓▓▌ preact-cli 2.2.1
▐▓▓ ▒▓▄▄▀▓▓▀ ▄▓▓ ▓▓▌
▐▓▌▐▓▓▓▀▀▄▄▀▀▓▓▓▌▐▓▌
  ▀█▄▄▒▓▓▓▓▓▓▒▄▄▒▀
      ▀▓▓▓▓▓▓▀▀
         ▀▀
For help with a specific command, enter:
  preact help [command]

Commands:
  create [template] [dest]  Create a new application.
  build [src] [dest]        Create a production build in build/
  watch [src]               Start a development live-reload server.
  serve [dir]               Start an HTTP2 static fileserver.
  list                      List all official templates

Options:
  -h, --help  Show help                                                [boolean]

Unknown argument: info
```

**Comment by RyanChristian4427** [comment]
Hi!

This is a known issue, resolved [here](https://github.com/preactjs/preact-cli/commit/2cba92cd74a59fba2b93bab1fabfa8a593408b11). The templates were all updated with new commands, but the CLI still gives the old hint.

This will be fixed the next time the CLI is released, I assume.

**Comment by reznord** [comment]
Fixed in #1146
